extension/content.js

```diff
@@ -344,6 +344,21 @@ function addFilterCommentsByYou() {
 	}
 }
 
+function addProjectNewLink() {
+	const projectNewLinkId = 'refined-github-project-new-link';
+	const projectNewLink = `<a href="/${repoUrl}/projects/new" class="btn btn-sm" id="${projectNewLinkId}">Add a project</a>`;
+	if ($(`#${projectNewLinkId}`).length === 0) {
+		$('#projects-feature').siblings('p.note').after(projectNewLink);
```

Comment:
Okay, in this case I think it will be better to go with the CSS selector. Seems to be more consistent with other places where we try to achieve a similar effect.